plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakePageSink.java

```diff
@@ -498,6 +498,7 @@ private FileWriter createParquetFileWriter(Path path)
                     parquetWriterOptions,
                     identityMapping,
                     compressionCodecName,
+                    DateTimeZone.UTC,
```

Comment:
Just as a heads up, we do have `io.trino.plugin.deltalake.DeltaLakeConfig#getParquetTimeZone` for Delta as well.

Reply:
In Delta Lake context we don't want to use the INT96 timestamp encoding which makes use of time zone. However, I've now made changes to send the value from DeltaLakeConfig here.